be/src/vec/functions/function_date_or_datetime_computation.h

```diff
@@ -995,4 +995,32 @@ class CurrentDateFunctionBuilder : public FunctionBuilderImpl {
     }
 };
 
+class FunctionTime : public IFunction {
+public:
+    static constexpr auto name = "time";
+    static FunctionPtr create() { return std::make_shared<FunctionTime>(); }
+    String get_name() const override { return name; }
+    size_t get_number_of_arguments() const override { return 1; }
+
+    DataTypePtr get_return_type_impl(const DataTypes& arguments) const override {
+        return std::make_shared<DataTypeTimeV2>();
+    }
+
+    Status execute_impl(FunctionContext* context, Block& block, const ColumnNumbers& arguments,
+                        uint32_t result, size_t input_rows_count) const override {
+        DCHECK_EQ(arguments.size(), 1);
+        ColumnPtr col = block.get_by_position(arguments[0]).column;
+        const auto* arg = assert_cast<const ColumnUInt64*>(col.get());
+        auto res = ColumnFloat64::create(input_rows_count);
```

Comment:
we'd better not use auto here. but `ColumnFloat64::MutablePtr`. then people will know the L1017 it not a virtual function call